test/fn.spec.php

```diff
@@ -364,6 +372,13 @@ function() { yield from [1,2,3]; }
             expect($res)->equal(null);
         });
     });
+    describe('identity', function() {
+        docFn(identity::class);
+        it('Returns its only argument as is', function() {
+            expect(identity(1))->equal(1);
+            expect(identity("foo"))->equal("foo");
+        });
```

Comment:
OK, we just need to a comment that this is an alias of `id` and vice versa so that it's clear that these do the same exact thing.

Reply:
Since it wasn't documented, I had the impression that we were keeping `id` for bc only. Do you plan to document it and keep it on future (breaking) versions?

Reply:
Gotcha, ya, I do intend on keeping it, personally I do prefer the shorthand for `id` and `not`, plus i have several apps that use both of those fns.